docs/Creating-Components.md

```diff
@@ -0,0 +1,124 @@
+
+Once done, make a PR - this first step will need code review and UX QA.
+
+Example implementations below:
+
+### index.jsx
+```js
+import styled from 'styled-components';
+
+const Blockquote = styled.blockquote`
+  background: #f9f9f9;
+`;
+
+export default Blockquote;
+```
+
+### index.stories.jsx
+```js
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import Blockquote from './index';
+
+storiesOf('Blockquote', module).add('default - quote only', () => <Blockquote>To be or not to be, that is the question</Blockquote>);
+storiesOf('Blockquote', module).add('quote with cite', () => (
+    <Blockquote cite="William Shakespeare">To be or not to be, that is the question</Blockquote>
+));
+```
+
+## Part 2: Create the container component and integrate it
+Once your component front-end is merged, you'll need to integrate it into the renderer. Let's work from the lowest level to the highest level:
+
+### Add a test
+In your component subdirectory, add a `index.test.jsx` to take a "snapshot" so that we can protect against regressions to the component. If the structure of the component changes, the unit test suite will fail.
+
+```js
+import React from 'react';
+import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
+import Blockquote from './index';
+
+describe('Blockquote', () => {
+  shouldMatchSnapshot(
+    'should render correctly',
+    <Blockquote>To be or not to be, that is the question</Blockquote>,
+  );
+});
+```
+
+### Update the schema
+data/schema.yaml describes the Article API definition for web and apps. We need to make it aware of our new component.
+
+Your component is more than likely a new 'block', so you'll need to add it to the array of what blocks are allowed to look like:
+
+```yaml
+    blocks:
+      type: object
+      items:
+        oneOf:
+          - $ref: '#/components/schemas/altText'
+          ... etc ...
+          - $ref: '#/components/schemas/[your component name]'
+      minItems: 2
+```
+
+You'll also need to define the block subtype itself:
+
+```yaml
+    blockquote:
+      type: object
+      required:
+        - model
+        - type
+      properties:
+        model:
+          properties:
+            blocks:
+              $ref: '#/components/schemas/blocks'
+          type: object
+        type:
+          enum:
+            - blockquote
+          type: string
+```
+
+The schema check currently only happens on local data. Eventually, we hope to run the schema validator at runtime.
+
+### Update some local fixture data
+Pick a JSON file under `data/test/news/articles/[id].json`, and:
+
+1) add an example of your block somewhere in the `content.model.blocks` array.
+2) add your new component to the `blockTypes` array.
+
+Run `npm run dev` and you should see your component at your article of choice, eg http://localhost:7080/news/articles/c0000000027o
```

Comment:
Updating the link to be a valid one would be great http://localhost:7080/news/articles/c0000000001o